app/controllers/api/stateless/consents_controller.rb

```diff
@@ -11,6 +11,11 @@ def create
         render json: member
       end
 
+      def destroy
+        member = Member.find_by_email! params[:email]
+        member.update!(consented_at: nil)
```

Comment:
Should check if member is nil?

Reply:
fortunately `Member.find_by_email!` will raise `RecordNotFound`

Reply:
I see! 👍